src/capi/azext_capi/custom.py

```diff
@@ -623,6 +624,10 @@ def create_workload_cluster(  # pylint: disable=too-many-arguments,too-many-loca
     with Spinner(cmd, "Waiting for workload cluster nodes to be ready", "✓ Workload cluster is ready"):
         kubectl_helpers.wait_for_nodes(workload_cfg)
 
+    if wait_for_nodes:
+        with Spinner(cmd, "Waiting for all workload cluster nodes to be ready", "✓ Workload cluster is ready"):
```

Comment:
I believe there is a lint problem for this line `is-too-long` [ci](https://github.com/Azure/azure-capi-cli-extension/actions/runs/3341990182/jobs/5533748597)